packages/web-react/src/components/Button/README.md

```diff
@@ -42,6 +42,7 @@ import { RouterLink } from 'react-router-dom';
 | `elementType` | `ElementType`                                                                             | `'button'` | no       | Type of element                                      |
 | `isBlock`     | `bool`                                                                                    | `false`    | no       | Span the element to the full width of its parent     |
 | `isDisabled`  | `bool`                                                                                    | `false`    | no       | If true, Button is disabled                          |
+| `isLoading`   | `bool`                                                                                    | `false`    | no       | If true, Button is in a loading state                |
```

Comment:
It would be better to write something like `it shows loading indicator`. The button itself has no state, so this can be misunderstood.

Reply:
And also write down that this prop also activates the `disable`.